packages/nextra-theme-docs/src/mdx-components.tsx

```diff
@@ -195,8 +196,8 @@ const Summary = (props: ComponentProps<'summary'>): ReactElement => {
     <summary
       className={cn(
         'list-none cursor-pointer rounded p-1 outline-none transition-colors hover:bg-gray-100 dark:hover:bg-neutral-800',
-        "before:content-[''] before:inline-block before:transition-transform dark:before:invert",
-        '[[open]>&]:before:rotate-90 rtl:before:rotate-180'
+        "before:mr-1 before:content-[''] before:inline-block before:transition-transform dark:before:invert",
+        '[[data-expanded]>&]:before:rotate-90 rtl:before:rotate-180'
```

Comment:
does previous didn't work? 🧐

Reply:
This makes sure that the arrow rotation animation isn’t delayed like the collapse animation

Reply:
Nice catch, remember that saw this issue :)